src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_AutoSuggestBox.cs

```diff
@@ -339,11 +347,11 @@ public async Task When_Text_Changed_Sequence(bool waitBetweenActions)
 
 			await WindowHelper.WaitForIdle();
 
-#if __SKIA__
-			// skia is closer to what happens on WinUI. On WinUI, if there is no delay between changes, 
-			// AutoSuggestBox.TextChanged is fired once (but TextBox.TextChanged fires everytime)
+#if __SKIA__ // We want to test the behaviour of "typing individual characters in sequence", not setting the Text in one shot. The behaviour is currently only accurate on skia.
 			if (!waitBetweenActions)
 			{
+				// skia is closer to what happens on WinUI. On WinUI, if there is no delay between changes,
+				// AutoSuggestBox.TextChanged is fired once (but TextBox.TextChanged fires everytime)
 				expectations = new() { SuggestionChosen };
 			}
 #endif
```

Comment:
I just took a look at the test results, it seems that on ios,droid,wasm, we only get a single event happening:
```
Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: 
CollectionAssert.AreEquivalent failed. The number of elements in the collections do not match. Expected:<7>. Actual:<1>.
expectations[7]: SuggestionChosen,ProgrammaticChange,UserInput,SuggestionChosen,UserInput,ProgrammaticChange,SuggestionChosen; 
actual[1]: SuggestionChosen
```
so we can remove the __SKIA__, since the behavior can be expected everywhere else.